app/src/common/shared/com/igalia/wolvic/browser/SettingsStore.java

```diff
@@ -289,6 +289,21 @@ public void setTrackingProtectionLevel(int level) {
         mSettingsViewModel.setIsTrackingProtectionEnabled(level != WContentBlocking.EtpLevel.NONE);
     }
 
+    public static boolean getStartWithPassthroughDefault() {
```

Comment:
I find the method name a bit confusing. What about "shouldStartWithPassthrougEnabled" ?